README.md

```diff
@@ -1,53 +1,62 @@
-# OpenGloves UI
 
-### This repository contains the user interface for configuration of the OpenGloves OpenVR driver.  
-Find it here: https://github.com/LucidVR/opengloves-driver
+# OpenGloves UI [![Steam Badge]][Steam] [![Discord Badge]][Discord]
 
-This interface reads the configuration keys from `default.vrsettings` specified in `../resources/settings/default.vrsettings` (relative to `src-tauri/`) to fetch the settings available (OpenVR doesn't seem to be able to do this), then fetches the user configurated settings via OpenVR. The `default.vrsettings` file is for default settings only, and an OpenVR Driver will look for settings found in the `steamvr.vrsettings` file, not the `default.vrsettings` the driver provides.
+*This repository is the user interface of the* ***[OpenGloves]*** *OpenVR driver.*
```

Comment:
Maybe also add a note at the end of this sentence like 

*This repository is the <ins>user interface</ins> of the* ***[OpenGloves]*** *OpenVR driver.* (And not the driver itself).

Along with the underline, this will help new people (who might not be here for source code) understand the purpose of this repo.

Reply:
Markdown doesn't provide underline syntax
as it's often hard to read, while I could use
the html tags, I will just try to adjust it in a 
different way instead.